app/components/profile.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import { useAccount, useEnsName } from "wagmi";
+
+export const Profile = () => {
+  const { address } = useAccount();
+  const { data, error, status } = useEnsName({
+    address,
+  });
+  if (status === "pending") return <div>Loading ENS name ...</div>;
+  if (status === "error")
+    return <div>Error fetching ENS name: {error.message}</div>;
+
+  return <div>ENS name: {data}</div>;
+};
```

Comment:
Since this is going to always be deleted. Should we had this directly on `page.tsx` file?

Reply:
I think we can have it here, since it keeps the component folder and the example is well organized.